pkg/scm/gitlab/client_actioner_assign.go

```diff
@@ -94,3 +116,100 @@ func (c *Client) AssignReviewers(ctx context.Context, evalContext scm.EvalContex
 
 	return nil
 }
+
+// getBackstageOwners returns a list of GitLab-mapped Backstage owners for the
+func getBackstageOwners(ctx context.Context, backstageClient *backstage.Client) ([]scm.Actor, error) {
```

Comment:
[suggestion] Most of this function (except for lines 145, 146, and 204) have nothing to do with GitLab.  Perhaps this (and the Backstage client initialization on lines 103-117) could be moved into a separate Backstage-centric package?  We'd then pass in the search annotations as inputs to a function, which would return the users from which we could extract their GitLab ID annotations.

Refactoring could make this easier to test, too!

Reply:
refactored this, lmk what you think